metadata_backend/api/handlers.py

```diff
@@ -1062,3 +1089,24 @@ async def _extract_xml_upload(req: Request, extract_one: bool = False) -> List[T
             files.append((xml_content, schema_type))
             LOG.debug(f"processed file in {schema_type}")
     return sorted(files, key=lambda x: schema_types[x[1]]["priority"])
+
+
+def _header_links(url: str, page: int, size: int, total_objects: int) -> CIMultiDict[str]:
```

Comment:
any reason not to have this as part of `RestAPIHandler` class?

Reply:
Good point